libs/ledger-live-common/src/families/algorand/buildTransaction.ts

```diff
@@ -9,13 +9,13 @@ import {
   makeAssetTransferTxnWithSuggestedParams,
 } from "algosdk";
 
-import type { AlgorandAccount, Transaction } from "./types";
-
+import type { Transaction } from "./types";
+import type { Account } from "@ledgerhq/types-live";
 import { extractTokenId } from "./tokens";
 import { getTransactionParams } from "./api";
 
 export const buildTransactionPayload = async (
-  account: AlgorandAccount,
+  account: Account,
```

Comment:
same, in my humble opinion it's a good idea to keep the xxxAccount inside the families and only call `as` when it's needed

Reply:
Similar to the above comment, the "buildTransactionPayload" method is called by "signOperation" that is a common method that is used in the common part.

Reply:
Will be treated outside this PR